### ISSUE TITLE
Support benchmark building against criterion-1.1.0.0

### ISSUE THREAD

**Issue by DanBurton** [tibbe/unordered-containers#122]

**Comment by tibbe** [comment]
Is it just an issue of bumping bounds? Mind sending a PR? I'm traveling.

**Comment by DanBurton** [comment]
Could be. I'll give it a go.

**Comment by DanBurton** [comment]
Looks like this was already done in acb67e3. We're just waiting on a release or a hackage revision, then.

**Comment by tibbe** [comment]
Released 0.2.7.0.

**Comment by DanBurton** [comment]
Awesome, thanks!
